openmetrics/tests/test_openmetrics.py

```diff
@@ -147,6 +128,7 @@ def test_openmetrics_default_instance(aggregator):
     aggregator.assert_all_metrics_covered()
 
 
+@pytest.mark.usefixtures("poll_mock")
```

Comment:
Having to add `@pytest.mark.usefixtures("poll_mock")` so many times seems not great IMHO.

Maybe the integration tests can be moved in a separate file, like here: https://github.com/DataDog/integrations-core/tree/master/network/tests

Reply:
Moving it as a decorator instead of a function parameter was not mandatory as "poll_mock" could have stayed as an argument to the test functions, but moving it there made it more explicit to see what changes from one test to another.

Yet, having a separate file for the integration test is probably more future-prone, and will allow the use of a top level fixture usage. Going this direction, will update the PR